include/Util/BasicTypes.h

```diff
@@ -251,6 +266,38 @@ class SVFFunction : public SVFValue
         return this->exitBB;
     }
 
+    void getExitBlocksOfLoop(const BasicBlock* bb, Set<const BasicBlock*>& exitbbs) const
+    {
+        if (hasLoopInfo(bb))
+        {
+            const std::vector<const BasicBlock*> blocks = getLoopInfo(bb);
```

Comment:
A copy construction should be avoided.

const std::vector<const BasicBlock*>& blocks = getLoopInfo(bb);